cristin-project/src/test/java/no/unit/nva/cristin/projects/create/createProject.feature

```diff
@@ -76,18 +75,3 @@ Feature: API tests for Cristin Project retrieve and search
     And match response.status == 400
     And print response.detail
 
-  Scenario: Create returns status 201 Created when input is valid and persisted in crisin
```

Comment:
We don't need all these creation of projects on each run, we already have a couple so those should be more than enough